src/Tooling/Bit.Tooling.Templates/TodoTemplate/Api/wwwroot/swagger/swagger-utils.js

```diff
@@ -5,6 +5,9 @@ based on: https://www.codedesigntips.com/2021/06/28/swagger-ui-with-login-form-a
     window.addEventListener('load', () => setTimeout(initLoginForm, 0), false);
 })();
 
+const AUTHORIZATION_TOKEN_KEY = 'access_token';
+let accessTokenAge = 0;
```

Comment:
based on the usage of this variable, it's better to rename it to `accessTokenExpiresIn`